adapter/operations.go

```diff
@@ -23,22 +23,26 @@ const (
 	OperationServiceNameKey  = "serviceName"
 )
 
+// OperationRequest contains the request data from meshes.ApplyRuleRequest.
 type OperationRequest struct {
-	OperationName     string
-	Namespace         string
-	Username          string
-	CustomBody        string
-	IsDeleteOperation bool
-	OperationID       string
+	OperationName     string // The name of the operation. This is used as lookup key in the Operations map.
+	Namespace         string // The namespace to use in the environment, e.g. Kubernetes, where the operation is applied.
+	Username          string // User to execute operation as, if any.
+	CustomBody        string // Custom operation manifest, for OpCategory_CUSTOM.
+	IsDeleteOperation bool   // If true, the operation specified by OperationName, is reverted, i.e. all resources created are deleted.
```

Comment:
Currently, we have a small challenge in the /management UI in which the same description of an operation is shown for "apply" and for "delete" operations. This challenge is overcome when we do not include verbs in the description of the operation (e.g. "Install Istio 1.7.3" vs "Istio 1.7.3" - using the "+" or "trash" buttons as the action designator).

I'm thinking aloud here as we could alternatively look to provide two different strings to describe a given operation - a provisioning and a deprovisioning string. I don't know that this solution is any better than just ensuring that we don't include verbs in the operation name. Ultimately, I think the right answer here is to design the UI differently.

Reply:
I think the strings without verbs would give us a lot of power to extend its abilities. We can control that in the UI is what I suggest @leecalcote

Reply:
I'll add a note to that effect, e.g. `do not use verb in OperationName`.